src/Nethermind/Nethermind.Core/Specs/ReleaseSpecDecorator.cs

```diff
@@ -67,6 +67,8 @@ public class ReleaseSpecDecorator(IReleaseSpec spec) : IReleaseSpec
     public virtual bool IsEip4895Enabled => spec.IsEip4895Enabled;
     public virtual bool IsEip4844Enabled => spec.IsEip4844Enabled;
     public virtual bool IsEip4788Enabled => spec.IsEip4788Enabled;
+    public ulong TargetBlobCount { get; set; }
```

Comment:
Should we change it to TargetBlobCount => spec.TargetBlobCount

Reply:
+1